app/DynamicsAdapter/DynamicsAdapter.Web/MatchFound/MatchFoundController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace DynamicsAdapter.Web.MatchFound
+{
+    public class MatchFoundResponse
+    { 
+        public string SearchRequestId { get; set; }
+        public string SearchResult { get; set; }
+    }
+
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MatchFoundController : ControllerBase
+    {
+        private readonly ILogger<MatchFoundController> _logger;
+        public MatchFoundController(ILogger<MatchFoundController> logger)
+        {
+            _logger = logger;
+        }
+
+        //POST
+        [HttpPost]
+        public async Task<IActionResult> MatchFound(MatchFoundResponse response)
```

Comment:
what we want here is the id in the path like `MatchFound/{id}`, then your method can take (Guid id, string payload), you have to use the `route` attribute to define the id. can you also add the `[Produces]`, `[ProducesResponseType]` attributes

Reply:
sure